MekHQ/resources/mekhq/resources/ForceType.properties

```diff
@@ -28,10 +28,13 @@
 # Microsoft's "Game Content Usage Rules"
 # <https://www.xbox.com/en-US/developers/rules> and it is not endorsed by or
 # affiliated with Microsoft.
+# suppress inspection "UnusedProperty" for whole file
```

Comment:
Man, I wish I understood why some property files are detected as used but others aren't.  Probably the way they're loaded...